firmware/application/ui_navigation.hpp

```diff
@@ -131,17 +131,20 @@ class StatusTray : public View {
     StatusTray(const StatusTray&) = delete;
     StatusTray& operator=(const StatusTray&) = delete;
 
+    void add_button(ImageButton* child);
```

Comment:
Why not alternate every Widget? Is the problem that you can’t set BG color on all of them?

Reply:
Yes, that was the issue.  I would prefer to shade every widget but there was no set_background function for the SD Card widget.  I'm thinking instead of shading (or possibly in addition to shading) that perhaps the better solution would be to add an extra pixel between each widget when there's room on the screen.  Or maybe increase the width of the speaker widgets to 17 pixels vs 16.

Reply:
We could use a template for ‘add’ and require types to have a set_background. Go ahead and add it to the SD card widget. It’s fine to change.